packages/litexa-html/README.md

```diff
@@ -44,7 +44,119 @@ project_dir
         └── html
 ```
 
-## Adding 'mark' SSML Tags
+## Alexa Web API Interface
+
+Integrating your skill with the Alexa Web API will allow it to both send and receive data
+from your web app.
+
+### Skill Manifest
+
+Before your skill can send and receive data through the Alexa Web API, you will need to
+specify the interface in your skill's manifest. You will need to include
+`ALEXA_PRESENTATION_HTML` in your `skill.*` file to let Alexa know that you are using it.
+For example, your skill manifest might look something like this:
+
+```json
+{
+  "manifest": {
+    "apis": {
+      "custom": {
+        "interfaces": [
+          {
+            "type": "ALEXA_PRESENTATION_HTML"
+          }
+        ]
+  ...
```

Comment:
FYI: This breaks markup - would generally use `js` and a comment instead: `// ...`